<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Productos</class>
 <widget class="QWidget" name="Productos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border: none;
}
#dashboard{
	background-color: #eff9fe;
}
#cards QFrame{
	background-color: white;
	border-radius: 15px;
}
#frMain{
	background-color: white;
	border-radius: 15px;
}
QLineEdit{
	background: white;
	border: 1px solid black;
	border-radius: 3px;
}
QPushButton{
	padding: 5px;
	border-radius: 5px;
	color:white;
	background: #0d6efd;
}
QPushButton:hover{
	background: #1e7fff;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="main" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QFrame" name="frMain">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFrame" name="frameView">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QTableView" name="tableView">
                 <property name="sizeAdjustPolicy">
                  <enum>QAbstractScrollArea::AdjustToContents</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frameAdd">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignTop">
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="lblNombre">
                 <property name="text">
                  <string>Nombre - Descripción</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="leDescripcion"/>
               </item>
               <item>
                <widget class="QLabel" name="lblMarca">
                 <property name="text">
                  <string>Marca - Modelo</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="leMarca"/>
               </item>
               <item>
                <widget class="QLabel" name="lblPrecio">
                 <property name="text">
                  <string>Precio</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lePrecio"/>
               </item>
               <item>
                <widget class="QPushButton" name="btnCreate">
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string>Añadir</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../PySide-Projects/intrefaceIA/resources.qrc"/>
  <include location="../../AICountDrive/interfaces/resources.qrc"/>
 </resources>
 <connections/>
</ui>
